packages/material-ui-styles/src/withTheme.test.js

```diff
@@ -51,4 +53,66 @@ describe('withTheme', () => {
 
     assert.strictEqual(isMuiElement(<ThemedInput />, ['Input']), true);
   });
+
+  describe('refs', () => {
+    it('forwards ref to class components', () => {
+      // eslint-disable-next-line react/prefer-stateless-function
+      class TargetComponent extends React.Component {
+        render() {
+          return null;
+        }
+      }
+      const ThemedTarget = withTheme()(TargetComponent);
+
+      const ref = React.createRef();
+      mount(
+        <>
+          <ThemedTarget ref={ref} />
+        </>,
+      );
+
+      assert.instanceOf(ref.current, TargetComponent);
+    });
+
+    it('forwards refs to React.forwardRef types', () => {
+      const ThemedTarget = withTheme()(
+        React.forwardRef((props, ref) => <div {...props} ref={ref} />),
+      );
+
+      const ref = React.createRef();
+      mount(
+        <>
+          <ThemedTarget ref={ref} />
+        </>,
+      );
+
+      assert.strictEqual(ref.current.nodeName, 'DIV');
+    });
+
+    describe('innerRef', () => {
+      beforeEach(() => {
+        consoleErrorMock.spy();
+      });
+
+      afterEach(() => {
+        consoleErrorMock.reset();
+      });
+
+      it('is deprecated', () => {
+        const ThemedDiv = withTheme()('div');
+
+        mount(
+          <>
```

Comment:
Fragment?

Reply:
There was an issue with refs on root components when mounting in enzyme. I think I added a comment somewhere but this might've been on a local branch.

Reply:
@oliviertassinari I can't find the original issue unfortunately. However it does indeed break if I remove the fragment. `ref.current` will be `undefined`.